### PR TITLE
Add sizes to the date pickers

### PR DESCRIPTION
**Pull request by patricklafrance**
Issue: https://github.com/gsoft-inc/sg-orbit/issues/92

## What I did

- Added a size `prop` to `<DateRangePicker />` & `<SingleDatePicker />` components.
- The sizes are currently tiny (24), small (32px), medium (40px) and large (48px)
- Updated the `<InlinedSingleDatePicker />` to icons size to em instead of a fix pixels width / height to adapt to the size.

## How to test

`yarn start-sb` and nativate to the "Components/DatePickers"

### CODE REVIEW

**Comment by patricklafrance** [comment]
Current input default size (medium) are all 48px

image

image

image

**Comment by patricklafrance** [comment]
After further discussions medium (default) size will be 40px and we will release a breaking change.

**Comment by fraincs** [comment]
the font-size will also be updated:
> 
> tiny -> 14px
> small -> 16px
> medium -> 18px
> large -> 20px

large should be 18px

File: packages/react-components/components/date-picker/src/date-picker-textbox-input.jsx [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
+const SIZES_TO_DEFAULT_ICON = {
+    [TINY]: <CalendarIcon24 className="w4 h4 fill-marine-700" />,
+    [SMALL]: <CalendarIcon24 className="w5 h5 fill-marine-700" />,
+    [MEDIUM]: <CalendarIcon24 className="w6 h6 fill-marine-700" />,
+    [LARGE]: <CalendarIcon className="w7 h7 fill-marine-700" />
+};
+
+const SIZES_TO_DISABLED_ICON = {
+    [TINY]: <CalendarIcon24 className="w4 h4 fill-cloud-500" />,
+    [SMALL]: <CalendarIcon24 className="w5 h5 ffill-cloud-500" />,
```
Comment (fraincs) [inline packages/react-components/components/date-picker/src/date-picker-textbox-input.jsx]: double f ici

**Pull request merged**